<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetManikinWnd</class>
 <widget class="QDialog" name="SetManikinWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>942</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#la1,#la2,#la1_2,#la2_2,#la1_3,#countLa{
	font-family: PingFangSC-Regular;
	font-size: 20px;
	font-weight: normal;
	line-height: 70px;
	color: #21b0f2;
}
QLabel#Linela,#Linela_2,#VLa,#Linela_3,#Linela_4,#Linela_5,#VLa_2{
	background-color: rgba(33, 176, 242, 0.5);
}

QPushButton#saveBtn,#cancelBtn{
	border-style:solid;
	background-color:#21b0f2;
	color:white;
	font-family: PingFangSC-Regular;
	font-size: 18px;	
}
QPushButton#bpm_Btn1,#bpm_Btn2,#bpm_Btn3,#bpm_Btn4,#voiceBtn_on,#voiceBtn_OFF,#AFCanBtn,#AFNoBtn,#AFIgnoreBtn,#SuccessCPRBtn,#SuccessAFBtn{
	font-family: PingFangSC-Regular;
	font-size: 20px;
	font-weight: normal;
	line-height: 70px;
	color: #21b0f2;
}</string>
  </property>
  <widget class="QLabel" name="titleLa">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>940</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>940</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: PingFangSC-Regular;
font-size: 24px;
font-weight: normal;
line-height: 70px;
color: #21b0f2;
background-color:#373950;</string>
   </property>
   <property name="text">
    <string>     模拟人设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>884</x>
     <y>20</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
image:url(:/image/X.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="la1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>节拍器</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointLa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>4</width>
     <height>4</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>4</width>
     <height>4</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00c594;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Linela">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>456</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>456</width>
     <height>1</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="la2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>语音提示</string>
   </property>
  </widget>
  <widget class="QLabel" name="Linela_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>1</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VLa">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>1</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="la2_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>除颤次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="Linela_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>310</y>
     <width>300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="la1_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>能否除颤</string>
   </property>
  </widget>
  <widget class="QLabel" name="Linela_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>456</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointLa_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>4</width>
     <height>4</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>4</width>
     <height>4</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00c594;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="VLa_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>1</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="la1_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>成功条件</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointLa_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>4</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00c594;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Linela_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>830</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>620</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>620</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpm_Btn1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpm_Btn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>100bpm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpm_Btn3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>110bpm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpm_Btn4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>120bpm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceBtn_on">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceBtn_OFF">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AFCanBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AFNoBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AFIgnoreBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>忽略</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SuccessCPRBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>CPR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SuccessAFBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>除颤</string>
   </property>
  </widget>
  <widget class="QLabel" name="countLa">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>355</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>次</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
